x-pack/plugins/ml/public/application/components/data_grid/use_column_chart.tsx

```diff
@@ -120,11 +120,11 @@ export const isUnsupportedChartData = (arg: any): arg is UnsupportedChartData =>
   return arg.hasOwnProperty('type') && arg.type === 'unsupported';
 };
 
-type ChartDataItem = NumericDataItem | OrdinalDataItem;
+export type ChartDataItem = NumericDataItem | OrdinalDataItem;
 export type ChartData = NumericChartData | OrdinalChartData | UnsupportedChartData;
 
 type LegendText = string | JSX.Element;
-const getLegendText = (chartData: ChartData): LegendText => {
+export const getLegendText = (chartData: ChartData): LegendText => {
```

Comment:
The legend always says `top 20 of... ` but we are showing a max of 10 columns here:

![image](https://user-images.githubusercontent.com/7405507/102225498-330c3800-3edf-11eb-8fee-f391ecbcdf06.png)

Reply:
Fixed here https://github.com/elastic/kibana/pull/85726/commits/286a76b18e6f8c7563503c965e654a8f4eecdd46